docs/backend/api/README.md

```diff
@@ -0,0 +1,83 @@
+# API Reference
+
+## Devices
+
+### Configure Devices
+
+    POST /api/devices
+
+Creates the device configuration file.
+
+### Parameters
+#### Header Parameters
+
+
+Field | Required | Description
+--- | --- | ---
+Authorization | Y | Session token.
+
+#### Body Parameters
+
+Field | Required | Description
+--- | --- | ---
+ThingD | Y | The ThingD configuration data that's sent by the user.
+
+### Example
+#### Request
+```bash
+POST http://localhost:8080/api/devices
+```
+##### Request Body
+```json
+{
+    "thingd": {
+        "name": "Thing Name",
+        "modbusSlaveID": 1,
+        "modbusSlaveURL": "serial:///dev/ttyUSB0:115200,N,8,1",
```

Comment:
```suggestion
        "modbusSlaveURL": "tcp://localhost:1502",
```

Suggesting TCP address because it's more common in our scenarios.

Reply:
Done!